### PR TITLE
DIP-14: Posters

### PR DESCRIPTION
**Pull request by VMLVaske**

### CODE REVIEW

File: DIPs/DIP-14.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+Also, people can visit the presentation area in their own time and stay as long or as short as they want. 
+
+## Implementation
+This proposal has been implemented at a large number of other events, mainly in the scientific sector. 
+It is quite common, sometimes even celebrated (when the beer comes into play) and while being easy and cheap to implement, offers many good networking synergy effects. 
+
+## Operational Requirements & Ownership
+1. What actions are required to implement the proposal at Devcon?
+
+Offer the possibility to submit posters - either parallely during the call for "speaking slots" or specifically to those, whose talk submission got rejected.
```
Comment (ligi) [inline DIPs/DIP-14.md:40]:
```suggestion
Offer the possibility to submit posters - either parallelly during the call for "speaking slots" or specifically to those, whose talk submission got rejected. 
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Comment by ligi** [comment]
* [ ] Correct markdown (e.g. headers with ##, header surrounded by ---)
 * [ ] syntax check
 * [ ] filename is DIP-X.md (with X being the correct DIP number)
 * [ ] header contains (in this order)
   * [ ] DIP
   * [ ] Title
   * [ ] Status
   * [ ] Theme(s)
   * [ ] Tag(s)
   * [ ] Author(s)
   * [ ] Resources Required (optional)
   * [ ] Discussion-to (with link to devcon forum)
   * [ ] Created (in the format year-month-day e.g 2020-4-20)
 * [ ] contains these sections:
   * [ ] Abstract
   * [ ] Rationale
   * [ ] Implementation
   * [ ] Operational Requirements & Ownership
 * [ ] PR title prefixed with DIP-X

**Review by VMLVaske — COMMENTED** [review]
Changed spelling

**Comment by ligi** [comment]
Thanks. The last checkbox I am missing is the created header. Then editor-wise it is OK to merge.

**Comment by VMLVaske** [comment]
Sorry for the delay! Added the creation-header-column :D

File: DIPs/DIP-14.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+---
+DIP: 14
+Title: Poster Presentations
+Status: Draft
+Themes: Community Involvement
+Tags: Event Operations
+Authors: valerie.vaske@gmail.com
+Resources Required: Physical space at venue, Communication support
+Discussion: https://forum.devcon.org/t/dip-14-using-posters-as-a-medium-alongside-talks/174/2
+Created: 25.11.2020
```
Comment (ligi) [inline DIPs/DIP-14.md:10]:
```suggestion
Created: 2020-11-25
```

**Review by ligi — CHANGES_REQUESTED** [review]
Thanks & no worries.
Just bringing this to our standard format and then I would merge

**Review by ligi — APPROVED** [review]

**Pull request merged**